src/Libraries/Nop.Services/Catalog/ProductService.cs

```diff
@@ -1243,7 +1244,7 @@ public virtual async Task UpdateProductReviewTotalsAsync(Product product)
 
         var reviews = _productReviewRepository.Table
             .Where(r => r.ProductId == product.Id)
-            .ToAsyncEnumerable();
+            .AsAsyncEnumerable();
```

Comment:
`AsAsyncEnumerable` is provided by LinqToDb and does not merely wrap an `IEnumerable` into an `IAsyncEnumerable`.